version/version.go

```diff
@@ -1,4 +1,4 @@
 package version
 
 // VERSION ...
-const VERSION = "2.2.0"
+const VERSION = "2.3.0"
```

Comment:
since the previous version we just updated the README: https://github.com/bitrise-io/envman/compare/2.2.0...master
why did you bump the minor version?

Reply:
I was told that we always bump the minor version during the monthly release and only fixes cause a patch version bump.